README.md

```diff
@@ -12,22 +12,41 @@ Data-Disk | Uncompressed .ISO | ddrescue
 DVD | MKV | MakeMKV
 BluRay | MKV | MakeMKV
 
-**To properly detect optical disk types in a docker environment this script relies on makemkvcon output.**
+## Prerequistites
+### (1) Create the required directories, for example, in /home/yourusername. Do _not_ use sudo mkdir to achieve this. 
 
-MakeMKV is free while in Beta, but requires a valid license key. Ripper tries to fetch the latest free beta key on launch. Without a purchased license key Ripper may stop running at any time.
-
-To add your purchased license key to MakeMKV/Ripper add it to the `enter-your-key-then-rename-to.settings.conf` at `app_Key = "`**[ENTER KEY HERE]**`"` and rename the file to settings.conf.
-
-# Docker run
+```
+mkdir config rips
+```
+### (2) Find out the name(s) of the optical drive
+```
+lsscsi -g
+```
+For example, /dev/sr0 and /dev/sg0 are the two files that refer to a single optical drive. These names will be needed for the docker run command.  
+![lsscsi -g](https://github.com/emmakat/docker-ripper/blob/emmakat-patch-1/screenshots/lsscsi%20-g.png)
```

Comment:
wrong repo

Reply:
![lsscsi -g](https://user-images.githubusercontent.com/29872150/82831475-66b54100-9e6d-11ea-833b-dc9a90ace486.png)

Reply:
currently trying to figure out how to do this. I read that I can make comments on each change, upload the image. But there's no clear instructions on How do I fix the PR that contains images inside a folder (the folder doesn't exist in the receiving repo). I think the markdown would reference https://github.com/rix1337/screenshots/lsscsi-g.png but I'm not sure. Perhaps I just need to do another branch that has the correct settings then make a new PR?

Reply:
or would the md be  https://github.com/rix1337/blob/screenshots/lsscsi-g.png